tests/stdlib/tjsonmacro.nim

```diff
@@ -632,7 +632,18 @@ proc testJson() =
       doAssert false, "this should be unreachable"
     except KeyError:
       doAssert getCurrentExceptionMsg().contains ".member.list[2].value"
-
+  block:
```

Comment:
remove tests/stdlib/tjson_enum_index.nim which is now redundant (refs https://github.com/nim-lang/Nim/pull/16807/files#r564146394)